src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import CommentSection from '../CommentSection/CommentSection.js';
+import dummyData from '../dummy-data.js';
```

Comment:
You don't have to bring in dummy data because you are passing down the data you need through props